src/ONYXKEYS.ts

```diff
@@ -563,6 +563,9 @@ const ONYXKEYS = {
         /** Expensify cards settings */
         PRIVATE_EXPENSIFY_CARD_SETTINGS: 'private_expensifyCardSettings_',
 
+        /** Expensify cards bank account settings */
+        EXPENSIFY_CARD_BANK_ACCOUNT_SETTINGS: 'expensifyCardBankAccountSettings_',
+
```

Comment:
```suggestion
        /** Expensify cards bank account for a given workspace */
        EXPENSIFY_CARD_BANK_ACCOUNT: 'expensifyCardBankAccountSettings_',
```

I think EXPENSIFY_CARD_BANK_ACCOUNT_SETTINGS gives the incorrect impression that we are storing bank account settings. But all we're doing is tracking whether the bank account is loading and if it was successfully added.

Reply:
Yeah, make sense. Maybe we can use METADATA like REPORT_METADATA?

Reply:
Nice, yeah, that would be better! 👍🏼

Reply:
Updated.